04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -410,14 +816,410 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this case we queried an SDSS DR7 table at the TAP service run by GAVO in Heidelberg.  \n",
-    "\n",
-    "We can get the details of a particular profile by including the name of the profile as an argument in the <i>list_profiles</i> method:"
+    "In this case we queried an SDSS DR7 table at the TAP service run by GAVO in Heidelberg. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's ensure it has been saved to VOSpace:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "003918.fits,6_drz.fits,QuickResults.csv,cutout.fits,gavo_out.csv,newmags.csv,ps1.vot,public,results,tmp\n"
+     ]
+    }
+   ],
+   "source": [
+    "listing = sc.ls (name='vos://')\n",
```

Comment:
Some users have quite a number of files in the VOSpace. It would help to list them in the long format, i.e. I suggest:
```
listing = sc.ls (name='vos://',format='long')
print(listing)
```